### PR TITLE
Prokka annotation tutorial: removed unnecessary commands; added slides

### PR DESCRIPTION
**Pull request by AnnaSyme**
For GCC 2019 workshop, took out some extra commands in Prokka and JBrowse that aren't needed anymore; also added a few more annotation slides. Moved slide images up into a higher level "images" directory.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5,14 +5,14 @@
   "requires": true,
   "dependencies": {
```
Comment (bgruening) [inline package-lock.json]: @AnnaSyme can you remove this change?

File: package.json [hunk 2]
```diff
@@ -12,6 +12,6 @@
   },
   "homepage": "https://github.com/galaxyproject/training-material#readme",
   "dependencies": {
-    "decktape": "^2.9.1"
```
Comment (bgruening) [inline package.json]: can you also remove this change?

File: package-lock.json (hunk 1 above)
Comment (AnnaSyme) [reply package-lock.json]: thanks @bgruening  - do I just click delete for this file (and the next one) ?

**Comment by AnnaSyme** [comment]
thanks @erasche looks good!

**Comment by erasche** [comment]
Thanks so much @AnnaSyme!

**Pull request merged**